sound.js

```diff
@@ -230,6 +243,30 @@ Sound.registerHeadsetPlugChangeListener = function(headsetPluggedInListener) {
   }
 };
 
+Sound.prototype.registerOnPlayListener = function(onPlayListener) {
```

Comment:
- I strongly recommend to remove this method because of the following reason: the upstream of this library consistently uses callback interface to deliver the event regarding the transition of internal variables. This listener breaks their rule, so it might not be acceptable for the upstream.
- So, I think registering this listener in the constructor of Sound class might be a safer choice.